README.md

```diff
@@ -58,9 +75,9 @@ class Application extends React.Component {
 - `edgeHitWidth` (Number) - Edge distance on content view to open side menu, defaults to 60
 - `toleranceX` (Number) - X axis tolerance
 - `toleranceY` (Number) - Y axis tolerance
-- `disableGestures` (Bool) - Disable whether the menu can be opened with gestures or not
-- `onStartShouldSetResponderCapture` (Function) - Function that accepts event as an argument and specify if side-menu should react on the touch or not. Check https://facebook.github.io/react-native/docs/gesture-responder-system.html for more details
-- `onChange` (Function) - Callback on menu open/close. Is passed `isOpen` as an argument
+- `onSwipe` (Function) - Function that handles gestures, receives boolean argument indicating whether menu should be opened or not based on the drag. When not defined, gestures are disabled
+- `onStartShouldSetResponderCapture` (Function) - Function that accepts event as an argument and specify if side-menu should react on the touch or not. Check https://facebook.github.io/react-native/docs/gesture-responder-system.html for more details. Typically you would like to set `isOpen` value to the passed argument (see examples/Basic.js)
+- `onContentPress` (Function) - Function that handles content press when menu is opened. Typically you would set isOpen to false inside it (see examples/Basic.js). When not present, content is accessible when menu is opened and there's no `touchToClose` behavior
```

Comment:
> When there's no onContentPress - content is accessible and one can close menu with the same button he used to open it - just thinking it can save us few issues from people complaining they don't know how to close the menu

I need this exact thing right now. When i open the menu `this.state.isOpen` gets correctly but when i try to close the menu the sidemenu press hijacks the event so `this.state.isOpen` is never changed when you close.